src/components/CategoryForm/CategoryForm.tsx

```diff
@@ -0,0 +1,100 @@
+import { LoaderOverlay } from 'components/LoaderOverlay/LoaderOverlay';
+import { ReactElement } from 'react';
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faPen } from '@fortawesome/free-solid-svg-icons';
+import { useCategoryFormContext } from './CategoryFormContext';
+import { Button } from 'components/Button/Button';
+
+export const CategoryForm = (): ReactElement => {
+  const formContext = useCategoryFormContext();
+  const [formCategory, setFormCategory] = formContext.useFormCategory;
+
+  //Placeholder Data
+  interface IProduct {
+    id: number;
+    name: string;
+  }
+  const Products: IProduct[] = [
+    { id: 0, name: 'Burger' },
+    { id: 1, name: 'Chips' },
+    { id: 2, name: 'Salad' },
+  ];
+  //TODO: Get this data from backend instead
+
+  const ToggleProduct = (event: React.ChangeEvent<HTMLInputElement>, toggleProductId: number): void => {
```

Comment:
we should use the `camelCase` convention for variables and functions. Components use `UpperCamelCase` and can easily be confused